release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -538,9 +538,44 @@ Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to t
 [id="ocp-4-17-insights-operator-enhancements_{context}"]
 === Insights Operator
 
-* The Insights Operator now collects the `haproxy_exporter_server_threshold` metric. (link:https://issues.redhat.com/browse/OCPBUGS-36687[*OCPBUGS-36687*])
+// Engineering reference CCXDEV-12582
 
-* Previously, the Insights Operator gathered information about all Ingress Controller certificates, including their `NotBefore` and `NotAfter` dates. This data is now compiled into a `JSON` file located at `aggregated/ingress_controllers_certs.json` for easier monitoring of certificate validity across the cluster. (link:https://issues.redhat.com/browse/OCPBUGS-35727[*OCPBUGS-35727*])
+The Insights Operator now collects more {product-title} container log data from namespaces prefixed with either the  “openshift-” or “kube-” prefix, and generates recommendations much faster.
+Enhancements have also been made to give you more flexibility in how the data to be collected gets defined for your service.
+
+==== Rapid Recommendations
+
+This release introduces a new feature called Rapid Recommendations, which provides a more dynamic and version-independent mechanism for remotely configuring the rules that determine which data the Insights Operator collects.
+
+Rapid Recommendations builds on the existing conditional data gathering mechanism.
+The Insights Operator connects to a secure remote endpoint service running on `/console.redhat.com` to retrieve definitions that contain the rules for determining which container log messages to be filtered and collected by Red Hat.
+
+The conditional data-gathering definitions, also referred to as rules, get configured through an attribute named `conditionalGathererEndpoint` in the link:https://github.com/openshift/insights-operator/blob/master/config/pod.yaml[`pod.yml`] configuration file.
```

Comment:
As the customer/user (e.g. SRE administrator) doesn't configure or control this attribute, the Red Hat host does, `get` is more appropriate here. Stating `are` implies anyone can, which isn't quite true.

(I wondered then why we even have to mention it... but the SME confirmed we do because the endpoint differs and offers more flexibility and backwards compatibility for customers through their subscription architecture, which they don't always directly configure/control).